src/presentation/http/router/note.ts

```diff
@@ -85,13 +89,33 @@ const NoteRouter: FastifyPluginCallback<NoteRouterOptions> = (fastify, opts, don
       return reply.notFound('Note not found');
     }
 
+    /**
+     * Wrap note for public use
+     */
+    let settings: NoteSettingsPublic | null = null;
+
+    if (request.noteSettings) {
+      settings = {
+        id: request.noteSettings.id,
+        customHostname: request.noteSettings.customHostname,
+        isPublic: request.noteSettings.isPublic,
+      };
+    }
+    const notePublic: NotePublic = {
```

Comment:
We'll need to repeat this code at many places. Can we move it to some decorator?